src/Logikfabrik.Umbraco.Jet/Mappings/DataTypeDefinition.cs

```diff
@@ -25,6 +25,11 @@ public enum DataTypeDefinition
         ContentPicker = 1034,
 
         /// <summary>
+        /// Data type for new content picker.
+        /// </summary>
+        ContentPicker2 = 1046,
```

Comment:
I get the use of the 2, but I'm afraid it might get hard to keep track of changes in the long run; if there's new editors added/changed in the future. Can we come up with some other naming scheme? Say ContentPicker760, for the new content picker introduced in version 7.6.0. and so on?